mne/source_estimate.py

```diff
@@ -1366,6 +1367,41 @@ def plot(self, subject=None, surface='inflated', hemi='lh',
         return brain
 
     @verbose
+    def return_to_original_src(self, src_orig, subject_orig=None,
```

Comment:
`map_back_to_src` is more explicit because it makes it clear it must have been mapped forward in the first place